e2e/pages/StageOne/EmailVerificationPage.ts

```diff
@@ -0,0 +1,77 @@
+import { Locator, Page } from "@playwright/test";
+import { assertComponentsVisible } from "../../utils/visibilityUtils";
+import { assertErrorUtil, assertErrorsHidden } from "../../utils/errorUtils";
+
+export default class EmailVerificationPage {
+  readonly page: Page;
+  // text
+  readonly checkEmailHeading: Locator;
+  readonly emailSentMessage: Locator;
+  readonly text1: Locator;
```

Comment:
Same as comment on ConsentPage, if can't be more specific then use 'first', 'second' etc instead of 1,2